src/layers/lsh.h

```diff
@@ -7,9 +7,9 @@ namespace faiss {
 
 namespace marian {
 
-class LSH {  
+class LSH {
 public:
-  LSH(int k, int nbits) : k_{k}, nbits_{nbits} {
```

Comment:
I'm assuming this will be guarded before integration.

Reply:
Yes, let's not resolve this conversation until it has been done later on.